spec/controllers/branches_controller_spec.rb

```diff
@@ -129,6 +129,23 @@
       end
     end
 
+    context "with extra convergence branche and one non-convergance" do
```

Comment:
typo: "branche"

please move this context the bottom of the status_report describe. The other three contexts are all related and this breaks them up.